package.json

```diff
@@ -70,7 +70,7 @@
     "@types/url-parse": "^1.4.3",
     "@typescript-eslint/eslint-plugin": "^2.7.0",
     "@typescript-eslint/parser": "^2.7.0",
-    "axios": "^1.3.4",
+    "axios": "^1.6.0",
```

Comment:
do we actually use axios somewhere?

Reply:
In the linkchecker: https://github.com/aws-amplify/docs/blob/main/tasks/link-checker.js